### PR TITLE
Add authenticated config for Allscripts

### PR DESCRIPTION
**Pull request by erikwiffin**
* Use a refresh token strategy for now to avoid writing a webdriver.
* Also provide un-authenticated config to Cerner.
* Improve display output for failing tests.

### CODE REVIEW

File: behave.ini.dist-allscripts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[behave]
+logging_level=DEBUG
+logging_format=%(message)s
+
+[api]
+url=http://uat-fhir.followmyhealth.com/api/
+patient=8501fd7d-4f46-4eca-a5c0-a5fd0112fe85
+
+[auth]
+strategy=refresh_token
```
Comment (jmandel) [inline behave.ini.dist-allscripts]: That's a cute trick :-)

And it buys you a *little bit of time* at least.

File: testsuite/oauth/refresh_token.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    def revoke_access_token(self):
+        """ Request that the oAuth server revoke stored access token.
+
+        Doesn't unset our currently stored data.
+        """
+        post_data = {
+            'token': self.access_token,
+        }
+        response = requests.delete(self._config['token_url'],
```
Comment (jmandel) [inline testsuite/oauth/refresh_token.py]: This won't be a _general_ feature of a token endpoint, for what it's worth. Is it supported anywhere?

Comment (erikwiffin) [inline testsuite/oauth/refresh_token.py]: No one supports it. Should we just remove this test?

Comment (jmandel) [inline testsuite/oauth/refresh_token.py]: Seems wise :-)

**Pull request merged**